### PR TITLE
Dara/jax scripts

### PR DESCRIPTION
**Pull request by Daramfon10**
Added JAX run script and also README.md file to help with running JAX benchmarks

### CODE REVIEW

**Comment by Daramfon10** [comment]
@microsoft-github-policy-service agree

**Comment by JonShelley** [comment]
@Daramfon10 do the instructions point to the proper location of the container once it is released? Or has it already been released? Are you expecting any changes to these instructions before the GA date?

File: apps/AI/JAX/README.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+    #For large models, use BS_PER_GPU=32
+    T5_SIZE=large BS_PER_GPU=32  sbatch -N 1 run.slurm
+
+    #For XL models, use BS_PER_GPU=8
+    T5_SIZE=xl BS_PER_GPU=8 sbatch -N 1 run.slurm
+    ``` 
+## Reviewing the results
+1. Once your slurm job is completed, open the output log file and search for **[200]**.
+2. In the summary line, extract the **timing/seqs_per_second** value. That is the throughput which are measuring for the benchmarks.
```
Comment (JonShelley) [inline apps/AI/JAX/README.md:66]: That is the throughput number we report for the benchmarks.

**Review by JonShelley — COMMENTED** [review]
@Daramfon10 please address these comments and then I think it will be ready for merge

**Review by JonShelley — APPROVED** [review]
Looks good. Thanks for making the changes. 

Approved

**Pull request merged**